vrmlogger/entrypoint.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 
 mkdir -p /data/conf /data/db
+sed -i 's+>900</LogInterval>+>60</LogInterval>+g' /data/conf/settings.xml
+sed -i 's+>0</HasDcSystem>+>1</HasDcSystem>+g' /data/conf/settings.xml
```

Comment:
hmm I need to verify this, but afaik you can change these settings via MQTT (or dbus), which then get persisted in the `settings.xml` file. With this, these settings will always get overwritten when the container starts, which is not ideal if a user changes the defaults via MQTT.